axelar-chains-config/info/mainnet.json

```
@@ -112,6 +112,12 @@
           "deployer": "0x6f24A47Fc8AE5441Eb47EFfC3665e70e69Ac3F05",
           "implementation": "0xA852412D2d91fE05d790934B6E64C3C351fAB54f",
           "address": "0x83a93500d23Fbc3e82B410aD07A6a9F7A0670D66"
+        },
+        "AxelarTransceiver": {
+          "salt": "Transceiver 1.1.0",
```

```suggestion
          "salt": "AxelarTransceiver v1.1.0",
```
use this salt for redeployment?